kubernetes/cronjob.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: descheduler-cronjob
+  namespace: kube-system
+spec:
+  schedule: "*/2 * * * *"
+  concurrencyPolicy: "Forbid"
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          name: descheduler-pod
+          annotations:
+            scheduler.alpha.kubernetes.io/critical-pod: ""
+        spec:
+          containers:
+          - name: descheduler
+            image: docker.io/aveshagarwal/descheduler:0.9.0 # TODO: change to official gcr.io registry when ready
```

Comment:
I'm not sure if we want to use the image from `docker.io/aveshagarwal` or not. This could be changed to just `descheduler` which is how it was previously configured in the README.md example.

@damemi and @aveshagarwal what do you think?

Reply:
yeah i think if descheduler image is pushed to gcr registry, then gcr registry should be used.

Reply:
Yeah ... i'm going to figure out how to get the production container registry setup for the descheduler. I'll update the image here once that is done.

Reply:
I updated this with the official location of the descheduler container image in gcr. The official gcr location for the descheduler is almost setup.

I also bumped the version to `v0.10.0` because I think it would probably be good to do a release soon. Especially once the official gcr registry is all setup.

Reply:
The container image build automation is all setup and working now.